build.gradle

```diff
@@ -225,8 +225,14 @@ tasks.register("verifyVersions") {
  */
 
 boolean bwc_tests_enabled = true
+String bwc_tests_disabled_issue = ""
+
+if (Os.isFamily(Os.FAMILY_MAC)) {
+  bwc_tests_enabled = false
+  bwc_tests_disabled_issue = "https://github.com/opensearch-project/OpenSearch/issues/4173"
+}
+
 /* place an issue link here when committing bwc changes */
```

Comment:
This comment should move with the variable

Reply:
Updated, thanks.